spotbugs/src/main/java/edu/umd/cs/findbugs/filter/SourceMatcher.java

```diff
@@ -57,6 +57,18 @@ public boolean match(BugInstance bugInstance) {
         if (DEBUG) {
             System.out.println("Matching " + bugFileName + " with " + fileName + ", result = " + result);
```

Comment:
It is better to add your logic BEFORE this debug log, then you don't need to copy-and-paste it at Line 68~71.

Reply:
Reordered Code as suggested.